docs/topics/simulation_setup.rst

```diff
@@ -0,0 +1,182 @@
+.. _simulation_setup:
+
+########################
+Setting up path sampling
+########################
+
+*Or: What OpenPathSampling does not do for you*
+
+While OpenPathSampling does a lot to simplify the process of doing a path
+sampling simulation, it doesn't do everything for you. In particular,
+setting up a path sampling simulation can still be challenging.
+
+Just as configurational Monte Carlo requires an initial configuration and
+a description of the ensemble (e.g., setting a temperature in an :math:`NVT`
+simulation), path sampling requires an initial trajectory and a description
+of the path ensemble. However, both describing the ensemble and creating an
```

Comment:
Sampling the paths using the TPS MC algorithm then would lead to the required equlibrated path ensemble.

Reply:
I'm not entirely sure what you meant here, although I tried to add something. See new text. By "describe the ensemble" I'm trying to say something akin to setting N, V, and T for an NVT ensemble.